products/public-websites/post-mortems/arrow-styles-01-22-21.md

```diff
@@ -9,7 +9,7 @@ Status: Resolved
 ## What happened
 The Public Websites team was lended extra engineering support by a fellow VSA team, the Authenticated Experience, due to some extra bandwidth in the latter team's sprint. Issues were pulled from the Public Websites backlog, many of which describing changes to code that had not been visited in quite some time.
 
-An [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556) was assigned for a benefit to the website's accessibility on a common component - the "link teaser." This issue called for a change to the HTML written in a CMS template. An Authenticated Exp engineer opened a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15698) implementing the change, which was reviewed by Public Websites and then merged into the vets-website master branch. The vets-website daily deployment was delayed due to unrelated cirumstances, so this change was deployed at roughly 5:30 PM ET.
+An [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556) was assigned for a benefit to the website's accessibility on a common component - the "link teaser." This issue called for a change to the HTML written in a CMS template. An Authenticated Exp engineer opened a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15698) implementing the change, which was reviewed by Public Websites and then merged into the vets-website main branch. The vets-website daily deployment was delayed due to unrelated cirumstances, so this change was deployed at roughly 5:30 PM ET.
```

Comment:
```suggestion
An [issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556) was assigned for a benefit to the website's accessibility on a common component - the "link teaser." This issue called for a change to the HTML written in a CMS template. An Authenticated Exp engineer opened a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/15698) implementing the change, which was reviewed by Public Websites and then merged into the vets-website master branch. The vets-website daily deployment was delayed due to unrelated cirumstances, so this change was deployed at roughly 5:30 PM ET.
```
Since this is a postmortem, `master` is accurate